cogs/sniper.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, bot):
         self.edited_message_cache_lock = Lock()
         self.removed_reaction_cache = []
         self.removed_reaction_cache_lock = Lock()
+        self.last_snipe = datetime.now()
```

Comment:
maybe this should be a dict of recent successful snipe timestamps per channel (and have a corresponding lock)?
the way it is now, i could see it behaving weirdly if snipe attempts are happening concurrently across different channels - have u tested that?